pkgs/tools/archivers/pax/default.nix

```
@@ -9,6 +9,8 @@ stdenv.mkDerivation rec {
     sha256 = "1p18nxijh323f4i1s2pg7pcr0557xljl5avv8ll5s9nfr34r5j0w";
   };
 
+  nativeBuildInputs = lib.optional stdenv.isDarwin utmp;
```

```suggestion
  buildInputs = lib.optional stdenv.isDarwin utmp;
```
I think this is a header only library which we compile against https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/darwin/apple-sdk/default.nix#L253-L254

---

Fixed.